archetype-common/src/main/java/org/apache/maven/archetype/DefaultArchetypeManager.java

```diff
@@ -146,6 +146,7 @@ public ArchetypeCatalog getInternalCatalog() {
 
             return source.getArchetypeCatalog(null, null);
         } catch (ArchetypeDataSourceException e) {
+            getLogger().warn("failed to read catalog", e);
```

Comment:
There is a warning ... but in PR title we have `Report error`

Reply:
That is a fair point. My initial idea was to `logError()` and throw an exception. But I noticed we already did `logWarning()` instead for remote catalog. In a way it can make sense to continue with the build and warn because you one catalog might fail, but others might not when you use `-DarchetypeCatalog=local,internal`.

I wanted to match that behaviour, I'll update the PR and ticket title.

Reply:
As it is warning maybe we should not print whole stacktrace in warn level but print in debug
we ca do it like:

```
getLogger().warn("failed to read catalog: " + e.getMessage(), getLogger().isDebugEnabled() ? e : null);
```

what do you think?

Reply:
It leads to a cleaner output for sure, the downside is that is might not get read if there are many archetypes in the catalog as the warning can be pushed outside of the terminal window. A stack trace is easier to spot, but definitely not the best solution either. Though I doubt that you skip the build logs it if you expect certain archetypes to be there.